PMMain/Midas/MidasGUI/GestureSeqRecorder.cpp

```diff
@@ -511,7 +511,7 @@ void GestureSeqRecorder::updateGuiSequences()
             }
         }
     }
-    else
+    else if (activeSequences.size() > 0)
     {
```

Comment:
These 2 lines are to fix a crash without the Myo. Shouldn't be necessary, but they fixed it...